### PR TITLE
patch: Clarify env variables in Documentation

### PR DESCRIPTION
**Pull request by vipulgupta2048**
- Clarifies env variables, adds a table for it 
- Add more information on `organization` and `API key` config properties

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: documentation/quickstart-autokit.md [hunk 1]
```diff
@@ -77,21 +77,23 @@ module.exports = {
 }
 ```
 
-Ensure that the `organization` matches the username that owns the `balenaApiKey` - otherwise it will lead to authentication errors.
+Ensure that you fill all field in the `config.js` file before proceeding. Refer to the {@page Config.js Reference | Config.js reference} for more information
```
Comment (perezmeyer) [inline documentation/quickstart-autokit.md:80]: s/field/fields

File: documentation/quickstart-qemu.md [hunk 2]
```diff
@@ -47,20 +47,23 @@ module.exports = {
 };
 ```
 
-To provide values of environment variables easily, you can create a `.env` file in the root of the leviathan directory. Use the format below as boilerplate. No need to do this for this test run. 
+Ensure that you fill all field in the `config.js` file before proceeding. Refer to the {@page Config.js Reference | Config.js reference} for more information
```
Comment (perezmeyer) [inline documentation/quickstart-qemu.md:50]: Same here.

**Review by perezmeyer — COMMENTED** [review]
Just a couple of possible typos.

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Pull request merged**